src/Ledger/hs-src/test/UtxowSpec.hs

```diff
@@ -24,13 +25,15 @@ initParams = MkPParams
   , minUTxOValue = 0
   , poolDeposit = 10
   , emax = 10
-  , pv = (1, 0) }
+  , pv = (1, 0)
+  , coinsPerUTxOWord = 1
+  }
 
 initEnv :: UTxOEnv
 initEnv = MkUTxOEnv {slot = 0, pparams = initParams}
 
-ada :: Coin
-ada = 0
+-- ada :: Coin
```

Comment:
should this be removed?

Reply:
Yes, I think we can remove it.  I just commented it out when I noticed it wasn't being used, but didn't want to delete it in case someone left it there for a reason.

Reply:
removed